design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,86 @@
+# Implementing Authenticator Status Checks
+The authenticator status check allows operators to check if the authenticator is configured correctly. With this capability, 
+operators can make a RESTful request using the Conjur API and receive a response identifying if configuration was successful.
+If not, the operators receive the proper error identifying the step of failure. 
+The first authenticator to utilize this feature is Authn OpenID Connect. 
+
+_For a complete status check flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+
+This doc describes how to expand the status check functionality to other authenticators.
+For each authenticator, a suite of [general](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn_status_general.md) 
+checks are run followed by those specific to the authenticator.  
+
+General checks include:
+1. The requesting user has access to the authenticator status route webservice
+1. _The authenticator implements the status check_
+1. The account exists
+1. The webservice exists
+1. The authenticator is enabled in the ENV
+1. _Checks specific for the new authenticator_
+
+To properly implement the status check functionality for the new authenticator, you will need to expand on italicized checks 2 and 6. 
+As described in step 2, the status of an authenticator method is checked for implementation. Therefore, you will need to 
+create a new class in the structure of Authentication::<New_Authenticator>::Authenticator and add a status method like so:
+
+```ruby
+def status(account:, authenticator_name:, webservice:)
+  Authentication::<New_Authenticator>::ValidateStatus.new.(
+    account: account,
+      service_id: webservice.service_id
+  )
+end
+```
+
+This status method will call the ValidateStatus’s CommandClass and run the new authenticator-specific checks as described in step 6. This new 
+ValidateStatus CommandClass should be in the structure of Authentication::<New_Authenticator>::ValidateStatus and will be need to be in a format similar to:
+
+```ruby
+module Authentication
```

Comment:
this format doesn't really tell the story. i think it will be better to reference the reader to the oidc implementation for an example https://github.com/cyberark/conjur/blob/master/app/domain/authentication/authn_oidc/validate_status.rb

Reply:
adding example as a side note below